web-chat/src/App.tsx

```diff
@@ -12,6 +12,28 @@ import { WakuContext } from './WakuContext';
 import { ThemeProvider } from '@livechat/ui-kit';
 import { generate } from 'server-name-generator';
 
+const themes = {
+  Message: {
+    css: {
+      margin: '0em',
+      padding: '0em',
+      fontSize: '1.3em',
```

Comment:
If you look at discord and slack. The nick has the same size font that the message. Except that the nick is bold. Let's aim for that.

Reply:
![image](https://user-images.githubusercontent.com/66876548/116654912-65986800-a98a-11eb-873b-ddacb3004148.png)

My discord looks like this. To be honest, I don't really like it, but what do I know LOL

I can try to make the name bigger than the text but that might require hacking the React component itself if they are internally using the same text attributes (honestly I'm not sure)

I've used slack but I'm not at the moment; I seem to be using a lot of different apps that do similar things by necessity, which I don't particularly like.

We can also add Avatar images or (I believe) text avatars, and have a Room selector with the toolkit; I kinda like the very minimal design but I thought UNIX talk was a great interface for communication.

Reply:
So based on this you want to make the sender bigger than the messages as well; that should be easy enough, I see what I can do

Reply:
I think this is pretty close, we can add Avatars and change the formatDate function to return "Today" when appropriate etc but I think that's the next iteration.